microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -143,24 +143,24 @@ To drive WebView2 with Selenium and Microsoft Edge Driver:
     ```csharp
     static void Main(string[] args)
     {
-        EdgeOptions edgeOptions = new EdgeOptions();
+        EdgeOptions eo = new EdgeOptions();
```

Comment:
@guangyuexu - Could you please review the code in this article regarding: I changed "edgeOptions" to "eo".  For example, in the rendered page and/or .md source file, Find "edgeoptions".  

I clicked "Re-request review" by your name but it didn't work.

Reply:
the edgeOptions changes look good to me.